### PR TITLE
Update appveyor.yml

### PR DESCRIPTION
**Pull request by XVincentX**
Related to #298

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
 
+install:
+  - git submodule update --init --recursive  
+
 build_script:
-  - git submodule update --init --recursive
   - vcbuild.bat MSVC2013
+  
+before_test:
+  - npm install -g cucumber
```
Comment (zdne) [inline appveyor.yml]: We need a ruby cucumber not node.js cucumber

Comment (zdne) [inline appveyor.yml]: see https://github.com/apiaryio/snowcrash/wiki/Building-on-Windows

File: appveyor.yml [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
 
+install:
+  - git submodule update --init --recursive  
+
 build_script:
-  - git submodule update --init --recursive
   - vcbuild.bat MSVC2013
+  
+before_test:
+  - gem install cucumber
```
Comment (pksunkara) [inline appveyor.yml]: We don't need these since the integration tests has been removed from shared/mson branch and move to apiaryio/drafter

Comment (klokane) [inline appveyor.yml]: In master branch still required

Comment (pksunkara) [inline appveyor.yml]: Yes, but this is an improvement which we are trying to merge into shared/mson branch. We don't need to back port this to master branch.

Comment (klokane) [inline appveyor.yml]: Good to remember used practices on appveyor. Thanks @XVincentX

**Comment by pksunkara** [comment]
This was prematurely merged before @XVincentX could remove integration test command from appveyor. :disappointed: 

#302 should fix it.

**Comment by klokane** [comment]
Sorry I missed it is against __mson__ branch...

**Pull request merged**